docs/getting-started/quickstart.rst

```diff
@@ -9,17 +9,18 @@ Quickstart
 Installation using Anaconda
 ---------------------------
 
-The easiest and recommended way to install Gammapy is to install the Anaconda
+The easiest way to install Gammapy is to install the Anaconda
 distribution from https://www.anaconda.com/download/ and then to install
-Gammapy and it's dependencies by executing this command in a terminal:
+Gammapy and its dependencies by executing this command in a terminal:
 
 .. code-block:: bash
 
     conda install -c conda-forge gammapy
 
-
-Alternatively, we also provide an environment definition file, so you can
-get additional useful packages together with gammapy in an isolated environment:
+Though this is one line command is the standard way to install a software package using Anaconda, **we recommend to
+make use of an environment definition file** that we provide, so you can get additional useful packages together with
+Gammapy in a virtual isolated environment. In order to proceed in this way, just copy and paste in your terminal
+the two lines below:
```

Comment:
Maybe add a link here like "If you want to learn more about using virtual env, see [link to section]"